sites/cheerpx/src/content/docs/13-tutorials/setup-svelte.md

```diff
@@ -0,0 +1,130 @@
+---
+title: CheerpX with NPM
+description: Using CheerpX together with Svelte
+---
+
+This tutorial will explain how to create a application with Svelte that works together with CheerpX.
+
+## Install Svelte
+
+First create a new Svelte app by running:
+
+```bash
+npm create svelte@latest my-app
+```
+
+1. Select the Skeleton Project option.
+2. Enable Typescript if needed.
+3. There are no additional options required for this tutorial.
+
+Next, navigate to your project directory and install the dependencies:
+
+```bash
+cd my-app
+npm install
+npm run dev
+```
+
+At this point, the development server will run from another terminal in the background.
```

Comment:
> the development server will run from another terminal
No, the dev server will run in the current terminal. The user will need to open a new terminal to keep working.

I highly recommend trying to follow the tutorial step-by-step to make everything that is written is consistent with what you see on the screen.